Cargo.toml

```diff
@@ -55,6 +55,7 @@ libfuzzer-sys = "0.4"
 log = "0.4"
 miette = "5.10"
 thiserror = "1.0"
+nalgebra = { version = "0.33.0", features = ["rayon"] }
```

Comment:
Do we need all of its default features plus rayon?

Reply:
Yes. I just checked this.